pkg/deleter/deleter.go

```diff
@@ -387,3 +391,23 @@ func (c *CleanupStatusTracker) RemoveStatus(pvName string, isJob bool) (CleanupS
 	}
 	return c.ProcTable.RemoveEntry(pvName)
 }
+
+func (d *Deleter) GetCleanPVSymlinksFunc() func(pv *v1.PersistentVolume) error {
```

Comment:
Not sure if this is the right place for this function.

Reply:
It is not. Why are defining this function here?

Reply:
I was thinking local-static-provisioner can provide a set of functions to be performed before PV deletion. These functions can be fetched and used by projects that are using local-static-provisioner as a library. Symlink cleanup can be one of them.

Reply:
Will move this function to LSO code base

Reply:
https://github.com/openshift/local-storage-operator/pull/254